MCPixelArtConverter/ImageConverterAverage.cs

```diff
@@ -19,26 +19,29 @@ public ImageConverterAverage(Dictionary<MCBlockVariant, Bitmap> palette)
         {
             Parallel.ForEach(palette, kv =>
             {
-                ColorDouble averageColor = new ColorDouble(0, 0, 0, 0);
-                Bitmap bm = kv.Value;
-                if (bm == null)
-                    return;
-
-                for (int w = 0; w < bm.Width; w++)
+                if (kv.Key.Selected)
```

Comment:
I would write if (!kv.Key.Selected) return;
That prevents nesting one level deeper, which is easier to track

Reply:
Done.